lib/signal-list-view.coffee

```diff
@@ -29,6 +29,11 @@ class SignalListView extends SelectListView
 
         @wsKernelCommands = [
             {
+                name: 'Rename'
```

Comment:
For me `Rename session for` or something similar would be clearer. I find `Rename Python 3 kernel` a bit misleading.

Other than that everything works beautifully 👍